rally-domain/src/main/groovy/yakworks/rally/mail/services/MailMessageSender.groovy

```diff
@@ -38,8 +40,11 @@ class MailMessageSender {
         Result result
         try {
             mailTo = convertMailMessage(mailMessage)
-            // mailService.send should never throw ex and should return result
-            result = emailService.send(mailTo)
+            boolean valid = validateEmail(mailTo.to[0]) //this will throw a problem when validation fails.
+            if(valid) {
```

Comment:
this is kind of confusing right? what happens when its false or when would it be false?

Reply:
Fixed